charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -38,12 +38,13 @@ spec:
   containers:
 {{ toYaml .Values.containers | indent 4 }}
   {{- end }}
+    {{- if .Values.volumeMounts }}
+    volumeMounts:
```

Comment:
Could you please correct the indentation?

Reply:
done.

Reply:
done.

Reply:
@runzhliu i didnt mean to change the indent to 6. If you check how other values are added there is change in alignment.